declarations_site/catalog/elastic_models.py

```diff
@@ -99,80 +94,79 @@ def get_family_members(self):
                     yield member["family_name"]
 
 
-
-class Declaration(AbstractDeclaration):
+class Declaration(DocType, RelatedDeclarationsMixin):
```

Comment:
Just made a little test with it instead of `_all`. The latter is much better so I'm not sure if `ft_src` makes much sense for site search at its current state but made it analyzable anyway.